src/mixins/legal-api-mixin.ts

```diff
@@ -156,4 +164,31 @@ export default class LegalApiMixin extends Vue {
     }
     return toReturn
   }
+
+  /**
+   * Ensure consistent object structure for a registration application,
+   * whether it contains a Name Request or not, and whether it is an initial
+   * draft or it has been previously saved. Object merging does not
+   * work very well otherwise due to nested properties.
```

Comment:
I wouldn't feel bad using lodash functions to merge objects, if that makes our life easier.

Another possibility that may be easier is to declare an empty object (in the IF file) and then we can assign (deep copy) that when we need it.

Reply:
(if alterantives are not easier then don't worry about it)

Reply:
I'm ok with the current implementation although if we want to remove these methods we can definitely refactor.
I can't recall the reason for initial implementation, do you recall?

Reply:
Don't refactor unless it simplifies your life.

I think you implemented this originally, and it had to do with merging data into an "empty" object.